### DISCUSSION TITLE
BCPS github.io MASS BLOCK

### DISCUSSION THREAD

**Discussion by lunarr341** [sleepy-cat-1/sleepy-cat-landing#7] [General]
im aware of any github.io site is blocked by Lightspeed Filter Agent and there isnt much i can do as of now

**Comment by prescottc4** [comment]
IMPORTANT WE HAVE MOVED TO GITLAB FOLLOWING THE MASSIVE GITHUB.IO BAN 
https://gitlab.com/sleepy-cat-1/weblanding
